x/evm/keeper/chainKeeper.go

```diff
@@ -30,13 +30,13 @@ var (
 	pendingDepositPrefix        = utils.KeyFromStr("pending_deposit")
 	confirmedDepositPrefix      = utils.KeyFromStr("confirmed_deposit")
 	burnedDepositPrefix         = utils.KeyFromStr("burned_deposit")
-	commandBatchPrefix          = utils.KeyFromStr("command_batch")
+	commandBatchPrefix          = utils.KeyFromStr("batched_commands")
 	commandPrefix               = utils.KeyFromStr("command")
 	burnerAddrPrefix            = utils.KeyFromStr("burnerAddr")
 	pendingTransferKeyPrefix    = utils.KeyFromStr("pending_transfer_key")
 	archivedTransferKeyPrefix   = utils.KeyFromStr("archived_transfer_key")
 
-	commandQueueName = "command_queue"
+	commandQueueName = "cmd_queue"
```

Comment:
why?

Reply:
because `commandPrefix`, `commandBatchPrefix`, `commandQueueName` all began with `command_`, and this would cause problems when trying to retrieve either a single command or the lsit of pending commands. I think this is the issue Milap reported on some weeks ago.